lib/src/clojure_lsp/handlers.clj

```diff
@@ -93,19 +93,18 @@
           {}
           work-done-token
           components)
-        (let [db @db*]
+        (let [db @db*
+              internal-uris (dep-graph/internal-uris db)]
           (when (settings/get db [:lint-project-files-after-startup?] true)
-            (f.diagnostic/publish-all-diagnostics! (-> db :settings :source-paths) components))
+            (f.diagnostic/publish-all-diagnostics! internal-uris components))
```

Comment:
This is the counter-part to the change to `publish-all-diagnostics!`. We publish all project findings, instead of re-calculating which files we might have findings for.

Reply:
Looks better indeed